zmon_worker_extras/check_plugins/nagios.py

```diff
@@ -8,8 +8,8 @@
 import shlex
 import subprocess32
 from functools import partial, wraps
-from zmon_worker_monitor.zmon_worker.errors import CheckError, NagiosError
 
+from zmon_worker_monitor.zmon_worker.errors import CheckError, NagiosError
```

Comment:
I guess NagiosError should be moved into the "extras" folder where the Nagios plugin resides